### PR TITLE
Web Share update for Chrome 61

### PR DESCRIPTION
**Pull request by samthor**

### CODE REVIEW

**Comment by petele** [comment]
Hey Sam - we already have https://developers.google.com/web/updates/2016/10/navigator-share, which I think cover the same things. Can we condense these articles down to a single one?

**Comment by samthor** [comment]
I've updated the old article instead (which works well as you're linking to it from #4997). PTAL!

File: src/content/en/updates/2016/10/navigator-share.md [hunk 1]
```diff
@@ -3,50 +3,42 @@ book_path: /web/updates/_book.yaml
 description: Sharing is caring. Chrome is running an Origin Trial to enable native sharing on the web.
 
 {# wf_published_on: 2016-10-20 #}
-{# wf_updated_on: 2016-10-20 #}
+{# wf_updated_on: 2017-09-06 #}
 {# wf_featured_image: /web/updates/images/generic/share.png #}
-{# wf_tags: chrome55,sharing,android,origintrials #}
-{# wf_featured_snippet: Sharing is caring. Chrome is running an Origin Trial to enable native sharing on the web called <code>Web Share</code> that allows websites to invoke the native sharing capabilities of the host platform. #}
+{# wf_tags: chrome55,chrome61,sharing,android,origintrials #}
+{# wf_featured_snippet: Sharing is caring. Web Share is now available in Chrome 61 for Android, and allows websites to invoke the native sharing capabilities of the host platform. #}
```
Comment (ebidel) [inline src/content/en/updates/2016/10/navigator-share.md]: add `{# wf_blink_components: Blink>WebShare #}`

Comment (samthor) [reply src/content/en/updates/2016/10/navigator-share.md]: Done

**Comment by samthor** [comment]
@petele for a review now

**Comment by PaulKinlan** [comment]
Techwise these all look good, so +1 and will merge assuming build bot doesn't find an issue.

**Pull request merged**